conf/general.yml-example

```diff
@@ -168,6 +168,12 @@ ALLOWED_COBRANDS:
   - cobrand2: 'hostname_substring2'
   - cobrand3
 
+# Control the loading of cobrand specific map asset layers.
+# If a cobrand is named here then `/web/cobrands/$cobrand/assets.js` will be
+# included in the JavaScript. You probably want to leave the blank.
+COBRAND_ASSETS:
```

Comment:
What's the preference for this over listing them in a `templates/web/fixmystreet.com/` specific template?

Reply:
None really, other than it seemed like a config thing so it should go in the config file. I guess there's an argument for putting it in a template that's updated as part of the adding the cobrand.

Reply:
I think I prefer a template, given it's not exactly something being used by other installations at present.